### PR TITLE
443-ps_cleanup

### PR DESCRIPTION
**Pull request by jacob-lee**
Intended to fix issue https://github.com/NYUCCL/psiTurk/issues/443

### CODE REVIEW

File: psiturk/psiturk_shell.py [hunk 1]
```diff
@@ -262,19 +240,19 @@ def do_config(self, arg):
         else:
             self.help_config()
 
-    config_commands = ('print', 'reload', 'help')
-
+    # TODO: this method does not appear to be used; remove?
```
Comment (deargle) [inline psiturk/psiturk_shell.py:243]: all of the `def complete_*` are used for tab completion by the shell

**Comment by deargle** [comment]
This looks good. Anything we can do to get pep8 to not complain about the def complete_ functions, which are actually used? Or is pep8 not complaining about that?

**Comment by jacob-lee** [comment]
Currently I only see the following:

PEP 8: E722 do not use bare 'except': This probably should be except KeyError
PEP 8: E722 do not use bare 'except': probably should catch ValueError and OverflowError
PEP 8: E265 block comment should start with '# ' -- this is commented out code
PEP 8: E265 block comment should start with '# ' -- this is commented out code
PEP 8 naming convention violation -- variable name is cmd_psiTurk (I missed fixing this one)

There were a number of declarations that didn't have PEP8's preferred line spaceing, and a lot of them used `'''comment'''` instead of `"""comment"""`. I fixed those.

I'll go ahead and fix those.

**Comment by jacob-lee** [comment]
This is from code in hit_create. I'm thinking it is reasonable to put a cap on the reward that might be entered; it would probably be appreciated should someone accidentally enter a very large number.

We could choose an arbitrary threshold (e.g. very liberal $10,000
```
        if m is None:
            self.poutput('*** reward must have format [dollars].[cents]')
            return
        try:
            reward = float(reward)
            if not math.isfinite(reward) or reward < 0 or reward > 10000:
                raise ValueError
        except (ValueError, OverflowError):
            self.poutput('*** reward must be in format [dollars].[cents]')
            return
```

or we could fetch there amt balance:

reward <= (self.amt_services_wrapper.amt_balance()).data  # might need to wrap a float around that amt_balance call ...

which would involve a call to amt ...

What do you think?

**Comment by deargle** [comment]
> I'm not sure if amt returns an error if you create a hit you don't have the balance for.

It does throw an error. The wrapper should already catch the error and return a result that has the exception bundled, and print the exception. I tried to move as much of the logic out of the shell and into the wrapper as I could, because I use the wrapper as an api of sorts for the web interface. Shell really is supposed to just do interactive shell-type things, and shunt rest of logic off to the wrapper. 

And the wrapper, in turn, doesn't do any actual amt calls -- that's in a different file. The wrapper is like a psiturk middleware abstraction of boto calls, interacts with psiturk db etc.

**Comment by jacob-lee** [comment]
Okay, in that case I'll skip the check inside the try block.

**Comment by jacob-lee** [comment]
Any further issues that need to be addressed before pulling?

**Comment by deargle** [comment]
I'm apparently terrible with git, so I opened a new PR using commits from this one plus one small tweak, but I labeled this as hacktoberfest-accepted which I think means it will count, lmk if otherwise.

And, my pep8 is complaining about different things, which means we should commit some kind of pep8 config file to the repo, for standardization. Would also be herculean to do a sweep through the whole project, but later.

**Pull request merged**